src/sentry/api/endpoints/organization_searches.py

```diff
@@ -24,14 +25,42 @@ def get(self, request, organization):
         :auth: required
 
         """
-        org_searches = Q(
-            Q(owner=request.user) | Q(owner__isnull=True),
-            ~Q(query__in=DEFAULT_SAVED_SEARCH_QUERIES),
-            project__in=self.get_projects(request, organization),
-        )
-        global_searches = Q(is_global=True)
-        saved_searches = SavedSearch.objects.filter(
-            org_searches | global_searches
-        ).order_by('name', 'project')
-
-        return Response(serialize(list(saved_searches), request.user))
+        if request.GET.get('use_org_level') == '1':
```

Comment:
Should this be based on the organization features instead of an explicit parameter? Once a user/org is moved over to org-searches do they need to be able to access their project searches?

Reply:
I generally prefer to use parameters like this, since it means I can test against an org that doesn't explicitly have the feature enabled.

I don't think there's any need for them to be able to access their project searches.

Reply:
Ok :+1: